api_tests/crossref/views/test_crossref_email_response.py

```diff
@@ -0,0 +1,160 @@
+import mock
+import pytest
+import hmac
+import hashlib
+
+from osf_tests import factories
+from website import settings
+
+
+@pytest.mark.django_db
+class TestCrossRefEmailResponse:
+
+    def create_mailgun_request_context(self, response_data):
```

Comment:
"request context" has a specific meaning in Django and Flask, so this is a bit of a misnomer. This is creating a request payload. See https://github.com/CenterForOpenScience/osf.io/pull/8344/files#r193214438